objectRecognitionSource/src/test/java/io/vantiq/extsrc/objectRecognition/neuralNet/TestYoloProcessor.java

```diff
@@ -69,6 +69,18 @@
     static final int TEST_IMAGE_HEIGHT = 375;
     static final int RESIZED_IMAGE_WIDTH = 100;
     static final int RESIZED_IMAGE_HEIGHT = 75;
+    
+    // For pre cropping tests
+    static final int PRECROP_TOP_LEFT_X_COORDINATE = 50;
+    static final int PRECROP_TOP_LEFT_Y_COORDINATE = 50;
+    static final int CROPPED_WIDTH = 200;
+    static final int CROPPED_HEIGHT = 150;
+    
+    // For pre cropping results test
+    static final int KEYBOARD_TOP_LEFT_X_COORDINATE = 120;
+    static final int KEYBOARD_TOP_LEFT_Y_COORDINATE = 250;
+    static final int KEYBOARD_CROPPED_WIDTH = 230;
+    static final int KEYBOARD_CROPPED_HEIGHT = 125;
```

Comment:
same comment as above :-(

Reply:
Intent here was that these constants are used in the precrop tests, so shouldn't they have the same PRECROP prefix as those on line 72ish.